deployment/kubernetes/charts/origin/templates/ipfs.ingress.yaml

```diff
@@ -16,6 +16,7 @@ metadata:
     nginx.ingress.kubernetes.io/enable-cors: "true"
     nginx.ingress.kubernetes.io/cors-allow-methods: "GET, PUT, POST"
     nginx.ingress.kubernetes.io/cors-allow-origin: "*"
+    nginx.ingress.kubernetes.io/limit-rps: "20"
```

Comment:
Seems a bit low. If we have 2 concurrent users on browse page, that's 2*12 listings = 24 listings to fetch. Each listing has its JSON blob hosted on ipfs and then at least 1 image. So that's about 2 * 12 * 2 = 48.

Reply:
These are per IP address, but you are right, maybe I'll make it 30.